app/src/androidTest/java/com/github/studydistractor/sdp/ui/LoginScreenTest.kt

```diff
@@ -36,6 +36,56 @@ class LoginScreenTest {
         }
     }
 
+    @Test
+    fun mainContainerExists() {

```

Comment:
Maybe we should discuss about standardize the Tags. We can discuss that in meeting